kerchunk/kerchunk-in-practice.ipynb

```diff
@@ -0,0 +1,2731 @@
+{
```

Comment:
<p>Not sure how we want to handle dependencies, but you do <code>import imagecodecs</code> in the first cell, so we assume it's already installed. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>